package-configuration/linux-cli/test_cli_install.py

```diff
@@ -51,10 +51,14 @@ def test_full_cli_install(self):
         finally:
             shutil.rmtree(tempdir)
 
-    def test_install_from_source(self):
+    def test_install_from_source_with_requirements(self):
         tempdir = tempfile.mkdtemp()
+        temp_requirements_file = tempfile.NamedTemporaryFile(delete=True)
+        with open(temp_requirements_file.name, 'w') as requirements_file:
```

Comment:
NamedTemporaryFile creates temp file in secure manner, it doesn't necessarily creates temp folder.
so your file can be in `/tmp`. usually user can delete files in `/tmp` only if it was created by him (because sticky bit is set on `/tmp`). so this is how `shutil` will possibly fail.
also, while `/tmp` can hold files that are critical for certain program and deleting them might crash it. so it's not nice. http://www.tldp.org/LDP/Linux-Filesystem-Hierarchy/html/tmp.html